src/numerical_method.py

```diff
@@ -171,13 +171,80 @@ def __init__(self) -> None:
 
 
 class PDEParams(Params):
-    def __init__(self) -> None:
+    def __init__(self, und_step: int, time_step: int, stock_min: int, stock_max: int, method: str) -> None:
```

Comment:
All params should have default values. You can either default it to None here and derive the proper value in the Pricer based on market data (e.g. stock_min, stock_max should depend on spot price and volatility).

Reply:
Fine to fix this later.